internal/util/paramtable/component_param.go

```diff
@@ -1396,6 +1398,16 @@ func (p *dataNodeConfig) initFlushInsertBufferSize() {
 	p.FlushInsertBufferSize = bs
 }
 
+func (p *dataNodeConfig) initFlushDeleteBufferSize() {
+	delBufferBytesStr := p.Base.LoadWithDefault2([]string{"DATA_NODE_DELETE_BUFFER_BYTES",
+		"datanode.flush.deleteBufBytes"}, "67108864")
```

Comment:
write it as 64 * 1024 * 1024, also please add datanode.flush.deleteBufBytes config to milvus.yaml